docs/reference/run.md

```diff
@@ -836,7 +836,7 @@ The container has unlimited memory which can cause the host to run out memory
 and require killing system processes to free memory. The `--oom-score-adj`
 parameter can be changed to select the priority of which containers will
 be killed when the system is out of memory, with negative scores making them
-less likely to be killed an positive more likely.
+less likely to be killed a positive more likely.
```

Comment:
I think this should be

```Markdown
with negative scores making them
less likely to be killed, and positive scores more likely.
```

Actually, could you move this to a separate PR, so that this can be cherry-picked into the docs separately? (sorry, nitpicking). I know there's a rewrite coming up for the reference/run.md, in which case it may be good to be able to cherry pick this as well so that the cherry-pick goes in cleanly